### ISSUE TITLE
Display better startup error messages

### ISSUE THREAD

**Issue by duncanspumpkin** [OpenRCT2/OpenRCT2#1648]
I've seen it quite often where people don't know how to get OpenRCT2 working because they have not installed RCT2 to the common address's or set it up incorrectly. We should really be showing an error message to the user when it fails rather than ending the program. (I know there is an error in cmd window but most people don't know to read that).

**Comment by daihakken** [comment]
maybe it can just implement like corsixTH, showing a message to let users to locate the folder...

**Comment by daihakken** [comment]
Like this:
![untitled-1](https://cloud.githubusercontent.com/assets/13048125/8766771/36fcb378-2e77-11e5-8148-b44a09e0029a.jpg)

**Comment by mrtnptrs** [comment]
That would be indeedly a nice idea and you could use the folder browser of the save/load window as the basic.... :)

**Comment by PFCKrutonium** [comment]
Dear Diary;
   Today I found out about CorsixTH.

**Comment by daihakken** [comment]
@PFCKrutonium omg, this. *ROFL*

**Comment by Superjustinbros** [comment]
I agree with this, that way I can finally have all these missing chunks errors gone from my copy of the game.

**Comment by IntelOrca** [comment]
Many people still report that running OpenRCT2 with incorrect configuration will open a console window and then close with no feedback or help. This has to be improved!

**Comment by Superjustinbros** [comment]
Yes!

**Comment by PFCKrutonium** [comment]
Perhaps on error, do the equivalent of VB's

```
Console.readKey()
```

That blocks the thread until a key is pressed with the Window selected.

**Comment by janisozaur** [comment]
See also https://github.com/OpenRCT2/OpenRCT2/issues/3190

**Comment by janisozaur** [comment]
As #3190 is marked for 0.0.5, should this be pushed back as well? There was already #3099 done to help with this.

**Comment by Gymnasiast** [comment]
OpenRCT2 has asked the user to provide a folder for a long time now, as well as looking in more places than it used to. Considering this solved.
